invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/CustomFieldSerializer.js

```diff
@@ -28,16 +28,25 @@ export class CustomFieldSerializer {
     if (customFields !== null) {
       for (const [key, value] of Object.entries(customFields)) {
         const isVocabularyField = this.vocabularyFields.includes(key);
+        const isCustomVocabulary = value.isCustomVocabulary;
```

Comment:
comment/minor: why do we have to track separately if the value is coming from a custom vocabulary? also, what is a "custom vocabulary" compared to a regular vocabulary? Aren't they served under `/api/vocabularies/...`.

If it's about being "generic" vocabularies, I think we should use the same terminology, i.e. `isGenericVocabulary`.